docs/source/tutorials/arch_modeling/index.rst

```diff
@@ -10,3 +10,4 @@ Architecture Modeling
    quick_start
    user_defined_temp_tutorial
    open_cell_libraries_tutorial
+   Spypads_tutorial
```

Comment:
Can you use lower cases for naming, so that we keep uniformity for tutorial names?